frontends/web/src/components/devices/bitbox02/settings.tsx

```diff
@@ -17,7 +17,6 @@
 import { Component, h, RenderableProps } from 'preact';
 import { load } from '../../../decorators/load';
 import { translate, TranslateProps } from '../../../decorators/translate';
-import RandomNumber from '../../../routes/device/settings/components/randomnumber';
```

Comment:
delete the file too `frontends/web/routes/device/settings/components/randomnumber.jsx`

Reply:
Can't, it's also used by BB01.

Reply:
Just read the title now :) . All good then I guess. cc @douglasbakkum

Reply:
🥀